bin/ghost

```diff
@@ -4,11 +4,22 @@
 // provide a title to the process
 process.title = 'ghost';
 
+function startup() {
+    const yargs = require('yargs');
+    const bootstrap = require('../lib/bootstrap');
+    bootstrap.run(process.argv.slice(2), yargs);
+}
+
 // Check for CLI updates if not running `ghost run`
 if (process.argv.length > 2 && process.argv[2] !== 'run') {
-    require('../lib/utils/update-notifier');
-}
+    const updateCheck = require('../lib/utils/update-check');
+    updateCheck().then(() => startup()).catch((error) => {
```

Comment:
can this not be `updateCheck().then(startup)...`?

Reply:
yup - it definitely can be :)